remote/theme.json

```diff
@@ -102,6 +102,11 @@
 				"border": {
 					"color": "var(--wp--preset--color--primary)"
 				}
+			},
+			"core/tag-cloud": {
+				"typography": {
+					"textTransform": "uppercase"
```

Comment:
Not sure if this should be done here or in the template.

Reply:
I prefer it here, unless there's a reason not to have it apply to every block.

Reply:
I don't understand, they are not uppercase in the design:
<img width="889" alt="Screenshot 2022-02-25 at 09 26 55" src="https://user-images.githubusercontent.com/3593343/155681332-f87b167a-fde8-44ca-a9db-2148d4f9c85d.png">

The tags/terms block is the one that is uppercase:

<img width="1070" alt="Screenshot 2022-02-25 at 09 27 51" src="https://user-images.githubusercontent.com/3593343/155681443-c557b2df-7b7b-4b45-a752-4e90e9046b31.png">

Reply:
Oh sorry my bad

Reply:
Updated